File: Problems/Week_1/3.23_나머지(10430)/이승민/10430.cs
using System;

namespace 백준1
{
    class MainProgram
    {
        static void Main(string[] args)
        {
            string[] ABC = Console.ReadLine().Split(' ');

            int A = int.Parse(ABC[0]);
            int B = int.Parse(ABC[1]);
            int C = int.Parse(ABC[2]);

            Console.WriteLine(((A + B) % C).ToString());
            Console.WriteLine((((A % C) + (B % C)) % C).ToString());
            Console.WriteLine(((A*B) % C).ToString());
            Console.WriteLine((((A % C) * (B % C)) % C).ToString());

        }
    }
}

File: Problems/Week_1/3.23_나머지(10430)/박창영/10430.py
A, B, C = input().split()

A = int(A)
B = int(B)
C = int(C)

print((A+B)%C)
print(((A%C) + (B%C))%C)
print((A*B)%C)
print(((A%C)*(B%C))%C)